aspnetcore/blazor/security/blazor-web-app-with-windows-authentication.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Secure an ASP.NET Core Blazor Web App with Windows Authentication
+author: guardrex
+description: Learn how to secure a Blazor Web App with Windows Authentication.
+monikerRange: '>= aspnetcore-9.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/13/2025
+uid: blazor/security/blazor-web-app-windows-authentication
+---
+# Secure an ASP.NET Core Blazor Web App with Windows Authentication
+
+<!-- UPDATE 10.0 - Enable after release
+
+[!INCLUDE[](~/includes/not-latest-version-without-not-supported-content.md)]
+
+-->
+
+This article describes how to secure a Blazor Web App with [Windows Authentication](/windows-server/security/windows-authentication/windows-authentication-overview) using a sample app in the [`dotnet/blazor-samples` GitHub repository (.NET 9 or later)](https://github.com/dotnet/blazor-samples) ([how to download](xref:blazor/fundamentals/index#sample-apps)).
```

Comment:
I think you may need to update the sample-apps page to add this sample to the list.